calculated_policies/aws_copy_tags_ec2_instance_to_s3_bucket/main.tf

```diff
@@ -0,0 +1,37 @@
+resource "turbot_smart_folder" "s3_tags" {
+  title       = var.smart_folder_title
+  description = "Enables bucket to over ride tags from ec2 instance"
+  parent      = "tmod:@turbot/turbot#/"
+}
+
+resource "turbot_policy_setting" "s3_bucketTags" {
+  resource = turbot_smart_folder.s3_tags.id
+  type     = "tmod:@turbot/aws-s3#/policy/types/bucketTags"
+  value    = "Enforce: Set tags"
+}
+
+resource "turbot_policy_setting" "s3_bucketTagsTemplate" {
+  resource = turbot_smart_folder.s3_tags.id
+  type     = "tmod:@turbot/aws-s3#/policy/types/bucketTagsTemplate"
+  # GraphQL to pull instance tags
+  template_input = <<EOT
+{
+  resource(id: 190397275456727) {
```

Comment:
This is not the write GraphQL, as we have fixed a value to a resource. What we need to do it use the current resource. We can discuss this.